### DISCUSSION TITLE
Existing table or new tables in existing app do not show updated 1.0.90 settings

### DISCUSSION THREAD

**Discussion by gpjorgenson** [Budibase/budibase#5045] [Help]
We have updated to version 1.0.90. I went in to update an existing table design in an existing app from the previous version and it does not present the new settings for a table row. I tried adding a new table in that same app and it still showed the old settings for a row. I created a new app and generated screens and that new app did show the new settings. Is there a conversion needed for an existing app's table to get the upgraded row settings?
Note: we have budibase installed on our own server. We are using a postgres db on our own server.

**Comment by mjashanks** [comment]
Try this button - the one in blue:

![image](https://user-images.githubusercontent.com/3524181/159791818-a2e876f9-2320-4764-b27d-1596ff1255f0.png)

This pulls in the new client library

**Reply by gpjorgenson** [reply]
Thank you. That worked!

**Reply by kasgithub8** [reply]
> Try this button - the one in blue:
> 
> ![image](https://user-images.githubusercontent.com/3524181/159791818-a2e876f9-2320-4764-b27d-1596ff1255f0.png)
> 
> This pulls in the new client library

@mjashanks What if we don't have that button yet?

**Reply by mjashanks** [reply]
@kasgithub8 you probably need to update your self-hosted instance, to get the latest version of Budibase. Once you do that, you will be able to update your apps.
